testmodels/src/main/java/com/amplifyframework/testmodels/meeting/Meeting.java

```diff
@@ -31,7 +31,7 @@
     private final @ModelField(targetType="AWSDate") AWSDate date;
     private final @ModelField(targetType="AWSDateTime") AWSDateTime dateTime;
     private final @ModelField(targetType="AWSTime") AWSTime time;
-    private final @ModelField(targetType="AWSTimestamp") Date timestamp;
+    private final @ModelField(targetType="AWSTimestamp") AWSTimestamp timestamp;
```

Comment:
It would, yea.

Reply:
Is this for compat with existing integration tests, or are you thinking this should be the ultimate design?

On one hand, my thought was that `AWSTimestamp` should map directly to `java.util.Date` because:
 - The data itself can be represented completely by `java.util.Date`
 - Simpler developer experience to have a way to have a `java.util.Date` object on the model itself, since it eliminates the need to call a `getDate()` conversion method or something before they can actually use it.


On the other hand, having our own `AWSTimestamp` is more congruent with `AWSTime`, `AWSDateTime`, `AWSDate`.

I guess what I'm saying is I don't feel that strongly one way or the other.   What were your thoughts?

Reply:
To document our offline conversation, we agreed on these as the right trade-offs.

We should confirm with @drochetti, but believe that the developer has no path to create a `java.util.Date` in a non-model type. So, if our models _also_ don't create `java.util.Date`, they would not be part of the system.

This does allow us to re-visit what to do with `Date` in the future, too, I guess.